### PR TITLE
FRMW-214: deprecate rxFormOptionTable alias

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-214

### LGTMs
- [x] Dev LGTM
- [x] Dev LGTM
- [x] QE LGTM

---
Removed deprecated `rxFormOptionTable` alias.  Use `rxOptionTable` instead.

### CODE REVIEW

**Comment by 100stacks** [comment]
## Removed Deprecated `rxFormOptionTable`

### API Docs: Before
rxformoptiontable 2016-04-14 15-16-53

rxoptiontable 2016-04-14 15-17-31

rxformutils 2016-04-14 15-20-38

### API Docs: After (removed `rxFormOptionTable` references) 
rxoptiontable 2016-04-14 15-18-28

rxformutils 2016-04-14 15-21-40

**Comment by Droogans** [comment]
@100stacks can you do me a favor and drop https://github.com/rackerlabs/encore-ui/blob/85873d2d43b5c9335306096068bacd10f71e8524/src/components/rxOptionTable/scripts/rxOptionTable.page.js#L608-L634 along with these other changes while we're at it?

**Comment by 100stacks** [comment]
@Droogans @CITguy,

File: src/components/rxOptionTable/scripts/rxOptionTable.page.js [hunk 1]
```diff
@@ -604,31 +604,3 @@ exports.rxOptionTable = {
...
-    /**
-     * @deprecated
-     * @function
-     * @memberof rxOptionFormTable
-     * @returns {rxOptionTable}
-     * @description
-     * **DEPRECATED**: Please use {@link rxOptionTable.generateAccessor} as a stand-in replacement.
-     */
-    generateAccessor: exports.rxOptionTable.generateAccessor
-};
```
Comment (100stacks) [inline src/components/rxOptionTable/scripts/rxOptionTable.page.js]: @Droogans @CITguy, removed `rxOptionFormTable`.

Updated wiki:
https://github.com/rackerlabs/encore-ui/wiki/2.0-Migration-Guide#removed-aliases
https://github.com/rackerlabs/encore-ui/wiki/2.0-Migration-Guide#removed-encorerxoptionformtable-page-object

**Comment by parlarjb** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

**Pull request merged**